model/generate.xml

```diff
@@ -307,34 +307,36 @@
     <option name="height" shortcut="t" type="uint32_t" description="The block height."/>
   </command>
 
-  <command symbol="fetch-filters-node" category="ONLINE" output="compact_filter" network="true" description="Get compact filters from a single Bitcoin network node. The block count between height and hash must be less than 100.">
-    <option name="filter" shortcut="r" type="filter" description="The compact filter type, defaults to 'neutrino'." />
+  <command symbol="fetch-compact-filters-node" category="ONLINE" output="compact_filter" network="true" description="Retrieve compact filters via a single Bitcoin network node. The distance between provided height and hash must be strictly less than 100.">
     <option name="format" type="encoding" description="The output format. Options are 'info', 'json' and 'xml', defaults to 'info'." />
     <option name="host" shortcut="t" type="string" default="localhost" description="The IP address or DNS name of the node. Defaults to localhost." />
     <option name="port" type="uint16_t" default="8333" description="The IP port of the Bitcoin service on the node. Defaults to 8333, the standard for mainnet." />
+    <argument name="filter_type" shortcut="r" type="uint16_t" description="The compact filter type." />
     <argument name="height" type="uint32_t" description="The block height."/>
     <argument name="hash" type="hash256" stdin="true" description="The Base16 block hash. If not specified the hash is read from STDIN."/>
-    <define name="BX_FETCH_FILTERS_UNSUPPORTED_FILTER" value="The peer does not support the filter type." />
+    <define name="BX_BIP157_UNSUPPORTED" value="The peer does not indicate support for BIP157." />
+    <define name="BX_INVALID_FILTER_TYPE" value="The provided filter type exceeds encodable limits." />
```

Comment:
This violates naming conventions. The constant should be prefixed with `BX_FETCH_FILTERS_NODE_` (more below).